packages/transformers/css/src/CSSTransformer.js

```diff
@@ -21,6 +21,10 @@ export default new Transformer({
   },
 
   async parse({asset}) {
+    if (asset.meta.hasDependencies === false) {
```

Comment:
This is set by the Stylus transformer to indicate that it has already processed all dependencies, and that the CSS transformer can skip this asset completely. This is required because when stylus processes e.g. `url()` it replaces them with a dependency id to be filled in later. When the CSS transformer runs, it would pick that up and try to resolve a dependency for the id which obviously doesn't exist. Also, it's faster to do it this way since the resulting CSS doesn't need to be re-parsed.

Reply:
Can you put this in a comment above this line?